src/components/Modal/Modal.styled.jsx

```diff
@@ -14,8 +14,7 @@ export const Overlay = styled.div`
 export const ModalContainer = styled.div`
   width: 260px;
   max-width: 343px;
-  height: 260px;
-  min-height: 375px;
+  height: auto;
```

Comment:
what about the `max-height`, was not needed?

Reply:
I test it without this, because, the `height` depends of the elements height, and it isn't necessary

Reply:
I think `max-height` would be needed. For example, if one modal includes so much content, then the modal can overflow to the bottom of the screen, and how all views of `maps-project` never scroll bottom, this can look bad or weird

Reply:
@Katsu08  so, should I use `max-height`?

Reply:
Yes, I think